build.cake

```diff
@@ -13,6 +12,9 @@ var UNIT_TEST_ASSEMBLY = "nunit-project-loader.tests.dll";
 
 var target = Argument("target", "Default");
 var configuration = Argument("configuration", "Debug");
+var nugetVersion = Argument("nugetVersion", (string)null);
+var chocoVersion = Argument("chocoVersion", (string)null);
+var binaries = Argument("binaries", (string)null);
```

Comment:
That was how I did the first packages (in a separate repo) but I think we may want to be able to repackage for nuget as well as chocolatey. Also, one reason you repackage is because you forgot a binary in the original package. I guess it may have to be manual.